src/Images.jsx

```diff
@@ -98,10 +96,8 @@ class Images extends React.Component {
                     this.setState({
                         setImageRemoveErrorModal: false
                     });
-                    const idDel = reply.image ? reply.image : "";
-                    const oldImages = this.props.images;
-                    let newImages = oldImages.filter(elm => elm.Id !== idDel);
-                    this.props.updateImages(newImages);
+                    this.props.updateImagesAfterEvent();
+                    this.props.updateContainersAfterEvent();
```

Comment:
Here the updateContainersAfterEvent is needed, but in the previous case no.

Reply:
yes, you are right.